src/lib/settings/initialSettings.tsx

```diff
@@ -100,6 +101,16 @@ export class Setting<T = unknown> {
         : this._default
       : this._default
   }
+  /**
+   * Return whether we should acknowledge this value as "matching" the current
+   * value. If the current value is the inherited default, we should also consider the
+   * it matching if the fallback value is matching.
+   */
+  public isCurrent(level: SettingsLevel | 'default', valueToMatch: T): boolean {
```

Comment:
Could we maybe change the name to be a bit more concise? I have no suggestions unfortunately...